test/plugin.test.js

```diff
@@ -758,7 +758,8 @@ test('pluginTimeout', t => {
   })
 })
 
-test('pluginTimeout default', t => {
+test('pluginTimeout default', { skip: 1 }, t => {
+  // TODO without lolex works
```

Comment:
@mcollina do you have any suggestion why this test fails with lolex and works without it?

Reply:
What error is triggered? lolex does some internal mocking and replaces some globals so it's well possible that it does not work with lolex. However, it looks like a bug. What error are you getting?

Reply:
Simply the test ends and fails because the plan is not completed: no throws